src/dlgAboutDialog.cpp

```diff
@@ -51,7 +51,7 @@ dlgAboutDialog::dlgAboutDialog(QWidget* parent)
 
         bool isWithinSpace = false;
         while (!isWithinSpace) {
-            QFont font(qsl("DejaVu Serif"), fontSize, QFont::Bold | QFont::Serif | QFont::PreferMatch | QFont::PreferAntialias);
+            QFont font(qsl("Bitstream Vera Serif"), fontSize, QFont::Bold | QFont::Serif | QFont::PreferMatch | QFont::PreferAntialias);
```

Comment:
The font name is 'Bistream Vera Sans'!

Reply:
Nope, the one used in the *About* dialogue (and on the Splash images) **has serifs** - which are more "ornamental" and look better in some situations (provided the font is shown big enough, Serif fonts rely more on anti-aliasing at small sizes and don't always work so well then). To prove the fact that we are using the Serif fonts - see here from a separate PR I am working on:
![image](https://user-images.githubusercontent.com/6163092/229286035-01e91a16-290c-4653-89e4-7370b42bd3d3.png)

Also - note the `QFont::Serif` in the arguments to the font-matching - which will try to get a serif font if the specified one is not found.

Reply:
You are right.